PinClustering/PinClustering/Model/FuelLocation.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  FuelLocation.swift
+//  PinClustering
+//
+//  Created by Norbert Agoston on 08/09/16.
+//  Copyright © 2016 Norbert Agoston. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class FuelLocation: NSObject, MKAnnotation {
+    
+    var title: String? {
+        return info?["station_name"] as? String
+    }
+    
+    var subtitle: String? {
+        return info?["fuel_type_code"] as? String
+    }
+    
+    var coordinate: CLLocationCoordinate2D {
+        let locationInfo = info?["location"] as? [String : AnyObject]
+        let latitude = locationInfo?["latitude"] as! String
```

Comment:
try not to force unwrap anything. Use here the ?? unwrap method instead in the CLLocationCoordinate2D constructor.